masonry/src/contexts.rs

```diff
@@ -412,6 +423,17 @@ impl_context_method!(MutateCtx<'_>, EventCtx<'_>, LifeCycleCtx<'_>, {
         self.widget_state.needs_layout = true;
     }
 
+    // TODO - Document better
+    /// Request a [`compose`] pass.
+    ///
+    /// The compose pass is often cheaper than the layout pass, because it can only transform individual widgets' position.
```

Comment:
```suggestion
    /// The compose pass is often cheaper than the layout pass, because it can only transform individual widget's position.
```

(As I just see this)

Reply:
I think the old version is fine?

... you know what, let me look it up.

Reply:
I think the original is the correct form for "the position of each of multiple widgets". However,
```suggestion
    /// The compose pass is often cheaper than the layout pass, because it can only change the position of widgets.
```
might be clearer still